test/cppsuite/tests/run.cxx

```diff
@@ -101,19 +95,21 @@ run_test(const std::string &test_name, const std::string &config_name = "")
 int
 main(int argc, char *argv[])
 {
-    std::string cfg, config_name, test_name;
+    std::string cfg, config_filename, test_name, current_test_name;
     int64_t error_code = 0;
     const std::vector<std::string> all_tests = {"example_test", "poc_test"};
 
     /* Parse args
-     * -C   : Configuration. Cannot be used with -f.
-     * -f   : Filename that contains the configuration. Cannot be used with -C.
+     * -C   : Configuration. Cannot be used with -f. If no specific test is specified to be run, the
+     * same coniguration will be used for all existing tests.
+     * -f   : Filename that contains the configuration. Cannot be used with -C. If no specific test
+     * is specified to be run, the same coniguration will be used for all existing tests.
      * -l   : Trace level.
      * -t   : Test to run. All tests are run if not specified.
      */
     for (int i = 1; (i < argc) && (error_code == 0); ++i) {
```

Comment:
Can we print a usage string here if the user inputs incorrect options? I've wanted this myself when trying to figure out how to run our runner.